### DISCUSSION TITLE
How to use the discussion forum?

### DISCUSSION THREAD

**Discussion by matthewmccarthy11** [life-itself/community#1140] [Research]
Pasted below is an exchange between @martinprogrammer and myself. Anyone please feel free to respond to any of the points and continue the discussion on we might want to leverage a discussion board, specifically as it relates to the research group.

## Distillation of main points and questions

- It is suggested by Martin that we consider more deeply the motivation and expectations of the research group and the people in it and why everyone is involved, at least in part because of the lack of engagement or follow-through on the discussion board.

- It is suggested by Matthew that having a 'low-stakes' discussion board where some things fall through is natural, but that the research group should find a way to have a more steady 'production cycle' which often (but not always) might start with discussion board topics. Also, that the discussion board might also be a way to consider the 'production cycle' of the research group more broadly. 
 

- Why is everyone here (esp in the research group) and what do they care about? (Can we use the discussion board to help facilitate that?)
- When we start a topic, what are expectations?
- How might we consider the discussion board in terms of 'workflow' of the research group? How can we 'develop' posts that are there and 'make something' out of them? Is there something we can do in the research group (weekly or bi-weekly) to maintain and 'cultivate' it and have a more steady 'production cycle'?
- How can we consider the discussion board as one part of a dialogue along with the whatsapp and the wiki?

### Matthew

Hey Martin, curious if you have any thoughts about the forum and having tn research call next week about the forum? I have no problem initiating etc, just want to get a sense with where you are/how you feel about it, or if  you have any other comments about the group more generally

### Martin

Hi Matthew, my observation was that there’s a lack of engagement.  We talk about it, we say we’re interested in doing something, but we don’t embody what we talk about.  I deliberately started a number of topics (in the past, on the forum) and have had a very limited traction.  Then I tried engaging with other people’s topics and found that people were not even interested in taking their own ideas any further.  So, my questions is - do we need to self reflect on our own inhibitions and try to uncover what’s stopping us from being more productive, engaged and effective.  Another approach would be to survey all of the topics we have had on the forum and chart their progress.  Maybe even ask ourselves - when we start a topic - what are expectations?  It can’t be that we’re starting something because we have a momentary inspiration and then next moment we don’t care about its lives to such extent that we don’t check the forum for another month.  Then we suddenly become aware of it and send some sort of response that will be ignored by everyone for another month…. What is your opinion - do my observation have any sense?  

What ideas do you have regarding the resurrection of the forum?

We’ve had some very good posts by various people - but it wasn’t clear whether the point was to showcase their work and garner some approval or if they are looking for collaboration.  Maybe we should introduce a prefix to every topic or simply create non-ambiguous channels.  A year ago I was sending some feedback to Rufus and told him that people introducing themselves under Introductions sometimes wait for weeks for any acknowledgment.  Not realising what effect such  latency does to someone’s enthusiasm points out at a lack of awareness.

When we have people wanting to participate somehow in the larger context of the LifeItself - is there any idea who we are looking for, what roles are there to be filled in, how do we find out what are people’s strengths (capacities) and their commitment?   In other words - does LifeItself have any clue what it wants on the operational level?

I’m just putting out some provocative questions to hopefully stimulate some sort of discussion 

Obviously, feel free to share with anyone - this is not meant to be private on my side

### Matthew

Thanks for the comments and for opening up these questions, and I have a bit of a long follow up. I definitely see what you mean about a lack of following up. From my view, I think that that is a natural feature of a discussion board- some topics fall to the wayside, or there isn't much more engagement. But I think you are right, and similar to what you say, I think there was no 'application' or 'development', or the way I would put it is, the engagement stops at a certain point; and perhaps the innovation we could try and do is to integrate these discussions into the research group and 'get them walking' in some way- either by discussing how we can 'put them into action' or how we can discuss it as a group, or put some attention on it as a group together.

The discussion board, in my opinion, is where we can introduce and work with our unformed ideas, and if we consider what we are doing in terms of 'production' or workflow the discussion board might be seen as the 'raw materials' of what we want to do; and the problem, I think, is not having a method or production cycle to work with these raw materials. What does that method look like? Curious what your thoughts are here (about this general view and exploring that workflow)

So I think having a low stakes discussion board where some ideas stick and some fall away is fine; but I think the problem is that it can't only be that, and maybe what we can do is figure out ways to incorporate it into the research group in some way.

I'm curious first if you think this is a decent approach, and secondly, how this might happen, and here are some ideas on my end:

To have a 'review' each research meeting of some of the possible topics; in research meetings, to spend 15 or so minutes on a 'topic of focus' where we unpack the topic and discuss it in the group, and see if we want to do anything with it (and if we do this, then we can say earlier in the research group before the meetings). To have a better 'classification' system (or make a better use of the classifications on GitHub) on the discussion board, ie. when introducing something, if it is an idea for a potential project, we can classify it as such; if it is 'this is an interesting article' it can also be classified as such.

### Martin

What I’d be interested to know is what are the motivations of people who join LifeItself or who enter these discussions.  If we figure out those - we can maybe classify them and manage our expectations.  I can only say why I joined myself - it’s because I’ve got strong ideas about what concrete projects could be started in order to change our lives / society /world for the better.  So, I’m not there in order to wallow in this general sense of hopelessness - but I’m on the side of sense-making and action.

So, I’d need to understand why are other people there.

Then, it would be useful to know what are the actual objectives of the LifeItself.  Not in terms of which direction they are walking in, but do they have any ideal outcomes and some concrete ideas of what this better future would look like.

I’m getting this feeling that time passes and work is getting down, but I can’t see the progress.  I would say progress is people getting closer together, producing more and agreeing more about what we should be doing

But we, as a collective, don’t seem to be that close - outside being polite to each other 🤣

What are the expectations of the Github discussions and research group itself?  Is it more than a reading club or a study group?

So this engagement that I was on about is the glue of collaboration.  You need to encourage people to first say what their interests, motivations are and then to learn what are their capabilities.

In more than a year that I’ve been a part of the community - this is the first time someone (Rufus) asked if I’d be interested in exploring how to improve Discussions.  In my own projects - I have 1 to 2 with everyone I feel could be useful and find out what their strenghts are and then ask for commitment and introduce a system by which their output could be measured.

Is that something that is deliberately not done here?

So your “method of production process” is probably what I’m talking about 🙂

What, why, how - a certain protocol

but - it can’t be you and I discussing this in isolation - we’re not the ones running this….

**Comment by martinprogrammer** [comment]
<img width="1153" alt="image" src="https://github.com/user-attachments/assets/afa663a0-023e-4aae-bdd1-0bf334143104">

I could scrape the discussions and create AI summaries similar to this one.  Not sure how useful at the moment, plz comment if you have any ideas.

**Reply by farandfew** [reply]
Martin you said " It can’t be that we’re starting something because we have a momentary inspiration and then next moment we don’t care about its lives to such extent that we don’t check the forum for another month. Then we suddenly become aware of it and send some sort of response that will be ignored by everyone for another month…. "
I don't know if anyone else is whistling and looking at their toes here, but are you sure it can't be like that? Because that pretty much describes my entire life at the moment.

**Reply by martinprogrammer** [reply]
I think you must have recorded it because that must have been my exact words ;-)
I'm exactly the opposite, I never join any enterprise unless I'm totally motivated and completely committed.  That's why I would like to talk about individuals' capacities and capabilities and the commitment because that will determine the scope of what can be done.

**Reply by farandfew** [reply]
I copied and pasted from above.
I do not see myself as having joined an enterprise. As far as I'm concerned, the biggest predictor of commitment is what is being done.

**Comment by rufuspollock** [comment]
I did quite a bit of analysis in this earlier thread about our general needs and ways we could address them

https://github.com/orgs/life-itself/discussions/992

I also think that, to start, we can focus on the needs of the existing g research group and maybe the broader life itself WhatsApp community.

My sense, and speaking partly for myself, is that some needs are

- a place to introduce oneself (more substantively than on the channel)
- a place to share links to interesting articles and events and discuss them
- a place to share ideas for projects
- a place to easily create informal landing pages for events like hackathons or meetups
- a place to share early stage writings or thinking (or even more mature work)

**Reply by matthewmccarthy11** [reply]
Agree with the bullet points above, @martinprogrammer had also mentioned that we include a more substantive introduction opportunity in the onboarding process, where we share our interests more. 

Also, we decided at the end of last meeting as a group that 'starting from our interests' is the best way forward, and the best way to understand where everyone is coming from/why and figure out a common path. Over the next few weeks in the research group we have decided to experiment with an 'internal seminar series' (name not fixed, and it was suggested 'seminar is too formal', maybe some other name is better?) where one person takes the hour to outline their own point of view(s)/projects/expectations more clearly, as well as guide the discussion in the way they see fit, whether that be through asking questions or outlining material. 

Perhaps after we do a number of these, we can then consolidate what was said and have the in person research retreat? So as to suggest a potential date, end of Feb is pretty open for me.

**Reply by farandfew** [reply]
I said that I would 'tentatively' agree to do the first one, this Friday, and that I would confirm today. I can confirm.
However I do have some stuff going on this Friday which makes some minor logistical problems. I can sort out but if anyone else is really wanting to go and would like to swap weeks, feel free to suggest!

I think that what I have to talk about is all on the theme of "Relating to Nature."
Or rather: "'relationship to nature' in the middle of a mass extinction."
This could be the  title.

I would imagine doing a quite traditional 30 minute talk with slides and then open discussion. Preparing such a thing would help me get my thoughts in order, but it's really all about the discussion. I would be open to alternative suggestions, or just to changing the ratio and making it - say - a 20 minute talk. I think it is also testing the format of future 'seminars.'

Does this work for everyone?

**Reply by matthewmccarthy11** [reply]
That sounds great with me! I can send a message in the whatsapp about the format and vision of these kinds of meetings, I just wanted to avoid doing that today since there was another research thing happening with DDS. 

And whatever format is good for you, but slides might help things along for sure. Sounds like a great title though, feeling excited for it :)
